SWLOR.Game.Server/Feature/StatusEffectDefinition/ForceHealStatusEffectDefinition.cs

```diff
@@ -23,11 +24,11 @@ public class ForceHealStatusEffectDefinition : IStatusEffectListDefinition
 
         private void ApplyHeal(uint source, uint target, int amount)
         {
-            var wilBonus = GetAbilityModifier(AbilityType.Willpower, source) * 2;
+            var wilBonus = GetAbilityModifier(AbilityType.Willpower, source);
             if (wilBonus < 0)
                 wilBonus = 0;
 
-            amount += wilBonus;
+            amount += (amount / 5 * wilBonus) + Random.D10(wilBonus/2);
```

Comment:
Just checking you intend this, but this is going to divide your amount by 5 _before_ multiplying by the Will bonus. Might be intentional, just calling it out.

Reply:
I'll go over the math later and double check this isn't in error.